src/js/components/Widgets/ShowMoreButtons.jsx

```diff
@@ -5,9 +5,45 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'styled-components';
 import { renderLog } from '../../common/utils/logging';
+import TagManager from 'react-gtm-module';
+import VoterStore from '../../stores/VoterStore';
 
 
 class ShowMoreButtons extends React.Component {
+  handleClick=()=>{
```

Comment:
Please change the generic function name `handleClick` to a function name that tells other programmers what this function is doing. For example `sendDataToGTMAndNotifyParent`.